.github/workflows/test.yml

```diff
@@ -84,6 +84,8 @@ jobs:
           wget https://ampereaimodelzoo.s3.eu-central-1.amazonaws.com/aio_objdet_dataset.tar.gz > /dev/null 2>&1
           tar -xf aio_objdet_dataset.tar.gz > /dev/null
           
+          apt-get update && apt-get install -y ffmpeg
```

Comment:
Should be handled by setup_deb.sh . One of the goals of this test to make sure all deps are installed by it, so if something is missing it should be added there.